projects/kit/src/components/callout/components/callout/callout.component.html

```diff
@@ -1,15 +1,17 @@
-<div *ngIf="isShown$ | async" class="callout" [ngClass]="'callout_' + variant">
-  <div class="callout__container">
-    <div class="icon"><pupa-icon [name]="icons.get(variant)"></pupa-icon></div>
-    <div>
+<div
+  class="callout"
+  [ngClass]="'callout-' + kind"
+  [class.collapsed]="isCollapsed$ | async"
+  [class.has-header]="hasHeader$ | async"
```

Comment:
селекторы пишем по БЭМу
```suggestion
  [class.callout_has-header]="hasHeader$ | async"
```